internal/receiver/smartagentreceiver/output_test.go

```diff
@@ -264,7 +264,18 @@ type mockMetadataClient struct {
 	name                    string
 	receivedMetadataUpdates []*metadata.MetadataUpdate
 	receivedLogs            []pdata.Logs
-	componenttest.ExampleExporterConsumer
+}
+
+func (me *mockMetadataClient) Start(_ context.Context, _ component.Host) error {
+	panic("implement me")
+}
+
+func (me *mockMetadataClient) Shutdown(_ context.Context) error {
+	panic("implement me")
+}
+
+func (me *mockMetadataClient) ConsumeMetrics(_ context.Context, _ pdata.Metrics) error {
+	panic("implement me")
```

Comment:
If anyone can file an issue in the core about this use-case to be provided/facilitate in core would be appreciated.

Reply:
Also an investigation if this is required at all to implement that interface will be good